scripts/zones/Lebros_Cavern/mobs/Qiqirn_Ceramist.lua

```diff
@@ -1,16 +1,16 @@
 -----------------------------------
 -- Area: Lebros Cavern (Excavation Duty)
---  Mob: Qiqirn Ceramist
+-- Mob: Qiqirn Ceramist
 -----------------------------------
 local ID = require("scripts/zones/Lebros_Cavern/IDs")
 -----------------------------------
 local entity = {}
 
-entity.onMobSpawn = function(mob)
```

Comment:
Functions don't seem to be declared properly for table values, unless something changed?